pkgs/by-name/ka/kani/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  fetchpatch2,
+  makeBinaryWrapper,
+  patchelf,
+}:
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

Should be preferred nowadays, see #315337